docs/logs/customizing-the-sdk/Program.cs

```diff
@@ -0,0 +1,42 @@
+// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using Microsoft.Extensions.Logging;
+
+using OpenTelemetry.Logs;
+
+public class Program
+{
+    public static void Main()
+    {
+        using var loggerFactory = LoggerFactory.Create(builder =>
+        {
+            builder.AddConsole();
+            builder.AddOpenTelemetry(options =>
+            {
+                options.AddConsoleExporter();
+            });
+        });
+
+        var logger = loggerFactory.CreateLogger<Program>();
+
+        logger.LogInformation("Hello Information");
+        logger.LogWarning("Hello Warning");
+        logger.LogError("Hello Error");
+
+        loggerFactory.Dispose(); // flush
```

Comment:
This is already handled by the using statement?

Reply:
I got that from another OTel example, don't remember where though.
I agree it's covered by the using, will remove.

Reply:
I can't find the source, but it's in other projects I was using to learn OTel.
I verified the examples in THIS repo are correct.

I vaguely recall an issue where logs were missing because they weren't being flushed.
Regardless, I agree the `using` is the correct pattern.